Button-2: Center image caption on frontend

nielslange opened the pull request:
Fixes #1725 

<table>
<tr>
<td>Before:
<br><br>

![#1725-before](https://user-images.githubusercontent.com/3323310/71499871-753b0480-289d-11ea-8d3c-e59ceb9f6008.png)

</td>
<td>After:
<br><br>

![#1725-after](https://user-images.githubusercontent.com/3323310/71499878-7cfaa900-289d-11ea-9c88-a944645959ae.png)
</td>
</tr>
</table>

michaeldcain commented on button-2/blocks.css:
```diff
@@ -20,7 +20,10 @@ Description: Used to style Gutenberg blocks
 /* Captions */
 
-[class^="wp-block-"] figcaption {}
+[class^="wp-block-"] figcaption {
```
Let's go ahead and copy [the original caption styles](https://github.com/Automattic/themes/blob/master/button-2/style.css#L1916) for the theme here. And then update [the editor styles](https://github.com/Automattic/themes/blob/master/button-2/editor-blocks.css#L156) to reflect this.


```
font-family: Lora, Garamond, serif;
font-style: italic;
text-align: center;
```

nielslange replied:
I've addressed the requested changes in https://github.com/Automattic/themes/pull/1726/commits/d82c13be435af125f9140355f60e3aad73925170

nielslange commented:
> It looked like some extra commits snuck in from #1666 during the last push. I dropped them from the PR, and otherwise, this LGTM. Thanks!

@michaeldcain I reverted the rouge commits and only the affected files are part of this PR now. 😀

michaeldcain commented:
> I reverted the rouge commits and only the affected files are part of this PR now.

@nielslange: FWIW, you can also do this by [rebasing the branch and dropping the unwanted commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History). I used `git rebase -i origin/master` to do this, and then force pushed it.

michaeldcain commented:
This was deployed to WP.com in r55055-wpcom-themes
